resources/pv-snapshot-all

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+
```

Comment:
I feel like all of this python script should be rewritten and moved to the devops-tools as we have all our things that help in our testing stuff